Okta.Xamarin/Okta.Xamarin.Android/Assets/AboutAssets.txt

```diff
@@ -0,0 +1,19 @@
+Any raw assets you want to be deployed with your application can be placed in
```

Comment:
Is this file necessary?

Reply:
This file was added by the VS Xamarin forms project template.  Since I intend to use this project as the basis for a future VS template which extends the base template, it should be left in place so our version of the template can be treated as an extension to the base template.